tests/unit/db/vertica/test_vertica_db_driver.py

```diff
@@ -68,36 +68,14 @@ def test_has_table_false(self):
         self.assertEqual(False,
                          self.vertica_db_driver.has_table(mock_schema, mock_table))
 
-    def test_best_scheme_to_load_from(self):
-        self.assertEqual(self.vertica_db_driver.best_scheme_to_load_from(), 'file')
-
-    def test_load_from_fileobj(self):
-        mock_schema = Mock(name='schema')
-        mock_table = Mock(name='table')
-        mock_load_plan = Mock(name='load_plan')
-        mock_fileobj = Mock(name='fileobj')
-        out = self.vertica_db_driver.load_from_fileobj(schema=mock_schema,
-                                                       table=mock_table,
-                                                       load_plan=mock_load_plan,
-                                                       fileobj=mock_fileobj)
-        mock_vertica_loader = self.mock_VerticaLoader.return_value
-        mock_vertica_loader.load_from_fileobj.assert_called_with(fileobj=mock_fileobj,
-                                                                 load_plan=mock_load_plan,
-                                                                 schema=mock_schema,
-                                                                 table=mock_table)
-        self.assertEqual(out, None)
-
     def test_load_failure_exception(self):
+        # TODO: Fix this test to use loader()
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker